integration/integration_test.go

```diff
@@ -12,22 +12,21 @@ import (
 )
 
 var kubebenchImg = flag.String("kubebenchImg", "aquasec/kube-bench:latest", "kube-bench image used as part of this test")
+var timeout = flag.Duration("timeout", 10*time.Minute, "Test Timeout")
 
 func TestRunWithKind(t *testing.T) {
 	flag.Parse()
 	fmt.Printf("kube-bench Container Image: %s\n", *kubebenchImg)
-	timeout := time.Duration(10 * time.Minute)
-	ticker := time.Duration(2 * time.Second)
 
-	mustMatch := func(expFname, data string) {
+	mustMatch := func(tc *testing.T, expFname, data string) {
```

Comment:
I needed to pass the `tc` down to the function because using the enclosed variable `t` was causing the test to send a false positive for the current test like:
```
pod (kube-bench-zx2nk) - "Succeeded"
--- FAIL: TestRunWithKind (109.67s)
    integration_test.go:29: expected:
    ....
    ....
    --- PASS: TestRunWithKind/kube-bench (15.23s)
FAIL
FAIL    github.com/aquasecurity/kube-bench/integration  109.773s
FAIL
make: *** [integration-tests] Error 1
```

instead of 


```
-- FAIL: TestRunWithKind (87.64s)
    --- FAIL: TestRunWithKind/kube-bench (8.21s)
        integration_test.go:29: expected: 
        ...
        ...
FAIL
FAIL    github.com/aquasecurity/kube-bench/integration  87.695s
FAIL
make: *** [integration-tests] Error 1
```

Reply:
The root of this problem is that `mustMatch()` is outside the enclosure for the version of `t` that's actually used in the tests